.github/workflows/docker-validate.yml

```
@@ -0,0 +1,192 @@
+
+      # Start services with docker compose using healthchecks
+      - name: Start services with docker compose
+        run: |
+          # Apply platform setting and start services
+          DOCKER_DEFAULT_PLATFORM=linux/amd64 docker compose up -d
+
+      # Validate that all services are healthy
+      - name: Validate services are healthy
+        run: |
+          # Wait for all services to be healthy
+          timeout 180s bash -c "until [ \"\$(docker compose ps --format \"{{.Name}}:{{.Status}}\" | grep -c \"healthy\")\" -eq \"3\" ]; do 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)